cmd/contact-auditor/main.go

```diff
@@ -7,54 +7,52 @@ import (
 	"flag"
 	"fmt"
 	"io/ioutil"
+	"os"
 	"strings"
 	"time"
 
-	"github.com/jmhodges/clock"
+	"github.com/go-sql-driver/mysql"
 	"github.com/letsencrypt/boulder/cmd"
 	blog "github.com/letsencrypt/boulder/log"
 	"github.com/letsencrypt/boulder/policy"
+	"github.com/letsencrypt/boulder/test/vars"
```

Comment:
Looks like you've pulled in the `test/vars` package to this non-testing code. That's not right - we should still be pulling the DSN from a `dbConnectFile`.

Reply:
oh gosh, yeah, you're right